signature/policy_eval_simple_test.go

```diff
@@ -1,3 +1,6 @@
+//go:build cgo || containers_image_openpgp
```

Comment:
This contains things like “reject all images from this repo” and needs to work, so it needs to be tested, even if “simple signing”  is disabled.